sigopt/objects.py

```diff
@@ -51,12 +51,28 @@ def __eq__(self, other):
     )
 
 
-class Assignments(ApiObject):
-  def get(self, key):
-    return self._body.get(key)
+class _DictWrapper(ApiObject, dict):
```

Comment:
Should we restrict setting attributes on metadata and assignments without copying?